ospec.js

```diff
@@ -18,6 +22,7 @@ else window.o = m()
 			return new Assert(subject)
 		} else {
 			if (isRunning()) throw new Error("Test definitions and hooks shouldn't be nested. To group tests, use 'o.spec()'.")
+			if (start == null) start = Date.now()
```

Comment:
This is a bit hacky, it is a compromise:

Previously, the timer was started when `o.run()` was being called, which is totally inaccurate, since it doesn't measure the time taken to run the glob task and register all tests. The timer runs until o.report() is called. This lets one run and report several times in a single session with the timer starting as close as possible to the actual start of the testing session without requiring a new API.

Reply:
Better: add a method to disable the timer printout and then just handle it at the CLI level. That way you know the timer's accurate.